docs/how-tos/validate-nexus-file.md

```diff
@@ -0,0 +1,802 @@
+
+Therefore you have enter all required fields in the NeXus definition. The requirements are set by the community via workshops or at conferences. You can as well comment the NeXus definitions, to initiate or propose changes/additions. Go to the NeXus definition, and sign-up/log-in and give us some feedback (Red boxes in the image. Expand this panel on the left by clicking on the arrow symbol).  
+![image.png](<./attachments/9d748230d54a8059-image.png>)
+
+Oftentimes, there will be errors in a generated NeXus file (be it by hand or automatically): Typos, missing required concepts, missing attributes, using the incorrect datatype or format (e.g., array instad of list, float instead of integer, etc.). Therefore, a validation is required, to ensure that the data you want to share, is FAIR.
+
+The NeXus file is valid if it complies with the respective NeXus application definition.
+
+This validation is done by software.
+
+
+# Validation software
+
+There are right now three tools, which can be used for validation of NeXus files. All are different and have individual advantages or disadvantages:
+
+1. pynxtools 
+
+2. cnxvalidate
+
+3. punx
+
+Open software is usually shared on Github - There you find usually the most accurate information, as documentation sometimes lags behind. There you see a box with folders and files. Below is the content of the README.md file displayed. This usually shows instructions for installation and handling of the software.
+
+Here are the GitHub links for the thee software packages:
+
+[pynxtools](<https://github.com/FAIRmat-NFDI/pynxtools>)
+
+[cnxvalidate](<https://github.com/nexusformat/cnxvalidate>)
+
+[punx](<https://github.com/prjemian/punx>)
+
+In the following, each package and its capabilities is presented.
+
+### Operating systems
+
+Almost all PC users are used to Windows as operating system.
+
+A lot of software development is done on Linux as operating system.
+
+This is not a problem for big companies, but for smaller open software projects, which are often developed without funding, this is a problem.
+
+If you are used to Windows, consider setting up a Linux operating system to eliminate problems in the installation process and ensure compatibility.
+
+
+# 1. pynxtools - Python Nexus Tools
+
+This is a python package which is developed by the FAIRmat consortium.
+
+As a python package, this can be used on Linux and Windows systems.
+
+The package can be installed via pip. Therefore you need to have installed:
+
+1. python
+
+2. pip
+
+This tool has 3 command line functions:
+
+1. [dataconverter](<https://github.com/FAIRmat-NFDI/pynxtools/blob/master/src/pynxtools/dataconverter/README.md>)
+
+2. [read\_nexus](<https://github.com/FAIRmat-NFDI/pynxtools/blob/master/src/pynxtools/nexus/README.md>)
+
+3. [generate\_eln](<https://github.com/FAIRmat-NFDI/pynxtools/blob/master/src/pynxtools/eln_mapper/README.md>)
+
+4. [verify\_nexus](https://github.com/FAIRmat-NFDI/pynxtools/pull/333)
```

Comment:
```suggestion
```
We should not put these here, as this is changing (actually, you are naming 4 tools here already).

Reply:
yea, right.